src/SamplesApp/UITests.Shared/Windows_ApplicationModel/Appointments/AppointmentStore.xaml.cs

```diff
@@ -0,0 +1,92 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Uno.UI.Samples.Controls;
+using Windows.Data.Xml.Dom;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Notifications;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+using Windows.ApplicationModel.Appointments;
+
+namespace UITests.Windows_ApplicationModel.Appointments
+{
+	[SampleControlInfo("Windows_ApplicationModel.Appointments", "AppointmentStore")]
+
+	public sealed partial class AppointmentStore : Page
+	{
+
+		public AppointmentStore()
+		{
+			this.InitializeComponent();
+		}
+
+		private async void getCalendarEntry_Click(object sender, RoutedEventArgs e)
+		{
+			uiErrorMsg.Text = "";
+			uiOkMsg.Text = "";
+			uiSubject.Text = "";
+			uiStartTime.Text = "";
+
+			Windows.ApplicationModel.Appointments.AppointmentStore oCalendarStore;
+			try
+			{
+				oCalendarStore = await AppointmentManager.RequestStoreAsync(AppointmentStoreAccessType.AllCalendarsReadOnly);
```

Comment:
The sample app does not have the read sample app permission. Could you add it so the sample app can work for this?

Reply:
Sorry, my error. Done.